_ext/funkwhale.md

```diff
@@ -0,0 +1,14 @@
+---
+title: mopidy-funkwhale
+type: backend
+dev:
+  gitlab: https://dev.funkwhale.audio/funkwhale/mopidy/
+  circleci: true
+  codecov: true
```

Comment:
This is probably not correct, as I see a GitLab CI file in the repo.

Reply:
This is outside my scope of knowledges :-)

Reply:
Better to remove it than to leave incorrect information then :-)

Reply:
As said in OP, i asked for some help on this, as i did not wanted to remove relevant informations aswell :-)

Reply:
Deleted lines as suggested